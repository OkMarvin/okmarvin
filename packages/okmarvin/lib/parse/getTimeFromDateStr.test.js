const getTimeFromDateStr = require('./getTimeFromDateStr')
test('returns date', () => {
  expect(getTimeFromDateStr('2018-10-10')).toMatchSnapshot()
})
