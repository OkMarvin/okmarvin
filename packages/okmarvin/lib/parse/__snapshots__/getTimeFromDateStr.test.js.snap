// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns date 1`] = `1539129600000`;
